// ðŸ“‚ refError.js
function refError(message) {
  throw new ReferenceError(message);
}
